Sofa/Component/SolidMechanics/Spring/src/sofa/component/solidmechanics/spring/RestShapeSpringsForceField.h

```diff
@@ -59,17 +60,18 @@ class RestShapeSpringsForceField : public core::behavior::ForceField<DataTypes>
     typedef typename DataTypes::CPos CPos;
     typedef typename DataTypes::Deriv Deriv;
     typedef typename DataTypes::Real Real;
-    typedef type::vector< sofa::Index > VecIndex;
+    typedef type::vector< sofa::Index > SetIndexArray;
```

Comment:
Changed naming to match FixedConstraint.

Reply:
VecIndex was maybe more clear. FixedConstraint should be changed.